File: OLD_JULIA/src_describe/remove_missing.jl
#=
    Removes the missing elements from the array
    Counts the number of missing elements and adds it to describe_table[head]
=#
function remove_missing(numerical_values, head, describe_table)
    missing_elem = []
    for i in 1:length(numerical_values[head])
        if (ismissing(numerical_values[head][i]) == true)
            push!(missing_elem, i)
        end
    end
    count = 0
    for elem in sort(missing_elem, rev=true)
        splice!(numerical_values[head], elem)
        count += 1
    end
    push!(describe_table[head], count)
end

File: src_describe/remove_missing.py
"""
    Remove missing values from the dictionary
"""

def remove_missing(headers, values):
    for head in headers:
        i = 0
        while i < len(values[head]):
            if (values[head][i] == ""):
                values[head].pop(i)
                i = 0
            else:
                i += 1
                